lib/server-config.js

```diff
@@ -27,11 +27,7 @@ const baseUri = url.format({
 
 // The base URI provides a suitable value for development. Production should
 // configure this.
-const allowedOrigin = envArray(
-  process.env.ALLOWED_ORIGIN,
-  baseUri.replace(/\/$/, ''),
-  ','
-)
+const allowedOrigin = envArray(process.env.ALLOWED_ORIGIN, [address], ',')
```

Comment:
As I read this change, I'm sorta questioning it. I think the bug this is trying to fix may be elsewhere.